imagebuildah/build.go

```diff
@@ -548,6 +550,8 @@ func (b *Executor) Prepare(ctx context.Context, ib *imagebuilder.Builder, node *
 		Labels:     builder.Labels(),
 		Shell:      builder.Shell(),
 		StopSignal: builder.StopSignal(),
+		//OnBuild:    tomString,
+		OnBuild:    builder.GetOnBuild(),
```

Comment:
OK here's the weird stuff.  If I set OnBuild to tomString, most everything works.  The example that @clcollins sent us runs fine.  Unfortunately at this point we have only read the FROM line in the Dockerfile and the builder's OnBuild is not yet set.  The strange part here is if I use tomString for the value, the example still runs however when I do `buildah inspect {image}` the two OnBuild values (OCI and Docker) are both listed as null.

Reply:
This is my Catch-22 that I'm still trying to figure out..  At this point I've not yet processed (or at least I don't know how to access it) the value that was in the ONBUILD line in the Dockerfile.  If I can figure out what that value is somehow at this point, I can jam it into the 'OnBuild:" line here at 554 and then things work well.  

Thanks for the heads up on the OnBuild field being null in the inspect, I thought that wasn't supposed to be like that once I got this working.